docs/keto/reference/ory-permission-language.mdx

```diff
@@ -0,0 +1,324 @@
+---
+title: The "Ory Permission Language" Specification
+---
+
+Enforcing fine-grained permissions is a critical building block of mature technology solutions that protect privacy and identity
+in the information age. Several proprietary languages to represent permission already exist, namely the Most permissions are
+defined by developers who are likely familiar with Web technologies like JavaScript or Typescript. There is a need for a
+developer-friendly configuration language for permissions that has such a small learning curve that most developers can understand
```

Comment:
```suggestion
developer-friendly configuration language for permissions that has a small learning curve small enough so that most developers can understand
```